yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -388,14 +388,8 @@ export class Archiver extends EventEmitter implements ArchiveSource, Traceable {
     currentL1BlockNumber: bigint,
   ): Promise<{ provenBlockNumber: bigint }> {
     const localPendingBlockNumber = BigInt(await this.getBlockNumber());
-    const [
-      provenBlockNumber,
-      provenArchive,
-      pendingBlockNumber,
-      pendingArchive,
-      archiveForLocalPendingBlockNumber,
-      provenEpochNumber,
-    ] = await this.rollup.read.status([localPendingBlockNumber], { blockNumber: currentL1BlockNumber });
+    const [provenBlockNumber, provenArchive, pendingBlockNumber, pendingArchive, archiveForLocalPendingBlockNumber] =
```

Comment:
Not necessarily for this pr, just got thinking, why are we using the viem contract directly in the archiver when we are using a `RollupContract` a bunch elsewhere (know there are a lot of both things around and about). Probably something we should look into at some point.